test/tracer/gold.trc

```diff
@@ -0,0 +1,488 @@
+rm -Rf tracer.exe tracer.dis
```

Comment:
Is this file intended to be checked in?

lt appears to be a log of commands on MacOS.

If it's expected trace output, it will need to be different under Linux.